app/helpers/patients_helper.rb

```diff
@@ -19,6 +19,11 @@ def language_options
     standard_options + Config.find_or_create_by(config_key: 'language').options
   end
 
+  def referred_by_options
+    standard_options = [[nil]]
```

Comment:
Let's port over the existing list as our standard options, and let funds staple on an additional set if they want via the config.

```suggestion
    standard_options = [nil, 'Clinic', 'Crime victim advocacy center',
     "#{FUND} website or social media",
     'Domestic violence crisis/intervention org', 'Family member', 'Friend',
     'Google/Web search', 'Homeless shelter', 'Legal clinic', 'NAF', 'NNAF',
     'Other abortion fund', 'Previous patient', 'School',
     'Sexual assault crisis org', 'Youth outreach']
```